### ISSUE TITLE
Availability: Make Summary Holdings Headers normal text

### ISSUE THREAD

**Issue by nicktiberi** [psu-libraries/psulib_blacklight#991]
Per accessibility testing with Michelle/Ruth/Binky, the "Holdings Summary" headings in the blue sections of the tables are confusing, and it's frustrating to have so many extra heading tags in the document because it makes navigation more difficult. It would be better to just have that text be a `<span>` or something with a `h6` style.

From:

```
<h6>{locationName}: Holdings Summary</h6>
```

To:
```
<span className="h6">{locationName}: Holdings Summary</span>
```
